[mpfr] Build error

**Host Environment**
- OS: x64 Ubuntu 20.04 (as WSL)
- Compiler: gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04) 

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install mpfr`

**Failure logs** 

 - Console output: 
> Building package mpfr[core]:x64-linux...
> -- Using cached mpfr-4.1.0.tar.xz.
> -- Extracting source /home/luis/workspace/vcpkg/downloads/mpfr-4.1.0.tar.xz
> -- Applying patch gmpd.patch
> -- Applying patch dll.patch
> -- Using source at /home/luis/workspace/vcpkg/buildtrees/mpfr/src/mpfr-4-8955d4bc66.clean
> -- Getting CMake variables for x64-linux-dbg
> -- Getting CMake variables for x64-linux-rel
> -- Generating configure for x64-linux
> -- Finished generating configure for x64-linux
> -- Configuring x64-linux-dbg
> -- Configuring x64-linux-rel
> -- Building x64-linux-dbg
> CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:155 (message):
>     Command failed: /usr/bin/make V=1 -j 9 -f Makefile all
>     Working Directory: /home/luis/workspace/vcpkg/buildtrees/mpfr/x64-linux-dbg/
>     See logs for more information:
>       /home/luis/workspace/vcpkg/buildtrees/mpfr/build-x64-linux-dbg-out.log
>       /home/luis/workspace/vcpkg/buildtrees/mpfr/build-x64-linux-dbg-err.log
> 
> Call Stack (most recent call first):
>   scripts/cmake/vcpkg_build_make.cmake:187 (vcpkg_execute_build_process)
>   scripts/cmake/vcpkg_install_make.cmake:26 (vcpkg_build_make)
>   ports/mpfr/portfile.cmake:23 (vcpkg_install_make)
>   scripts/ports.cmake:142 (include)
> 
> 
> Error: Building package mpfr:x64-linux failed with: BUILD_FAILED
> 

buildtrees/mpfr/autoconf-x64-linux-err.log 
> autoreconf: Entering directory `.'
> autoreconf: configure.ac: not using Gettext
> autoreconf: running: aclocal --force -I m4
> aclocal: warning: couldn't open directory 'm4': No such file or directory
> autoreconf: configure.ac: tracing
> autoreconf: running: libtoolize --copy --force
> autoreconf: running: /usr/bin/autoconf --force
> autoreconf: configure.ac: not using Autoheader
> autoreconf: running: automake --add-missing --copy --force-missing
> configure.ac:411: installing './compile'
> configure.ac:37: installing './missing'
> src/Makefile.am: installing './depcomp'
> autoreconf: Leaving directory 


**Additional context**
Previously had an error that required to install the autoconf-archive package

--- a/ports/mpfr/portfile.cmake
+++ b/ports/mpfr/portfile.cmake
@@ -1,3 +1,7 @@
+if (VCPKG_TARGET_IS_LINUX)
+    message(WARNING "${PORT} currently requires the following tools from the system package manager:\n    autoconf-archive\n    texinfo\nThese can be installed on Ubuntu systems via\n    sudo apt-get update -y\n    sudo apt-get install -y autoconf-archive texinfo\n")
+endif()
+
 set(VERSION 4.1.0)
 vcpkg_download_distfile(ARCHIVE
     URLS "http://www.mpfr.org/mpfr-${VERSION}/mpfr-${VERSION}.tar.xz" "https://ftp.gnu.org/gnu/mpfr/mpfr-${VERSION}.tar.xz"
@@ -15,7 +19,7 @@ vcpkg_extract_source_archive_ex(
 
 file(REMOVE_RECURSE "${SOURCE_PATH}/m4")
 vcpkg_configure_make(
-    SOURCE_PATH ${SOURCE_PATH}
+    SOURCE_PATH "${SOURCE_PATH}"
     AUTOCONFIG
     ADDITIONAL_MSYS_PACKAGES texinfo gettext autoconf-archive
 )
@@ -32,4 +36,4 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
 # Handle copyright
-file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

--- a/ports/mpfr/portfile.cmake
+++ b/ports/mpfr/portfile.cmake
@@ -1,5 +1,5 @@
 if (VCPKG_TARGET_IS_LINUX)
-    message(WARNING "${PORT} currently requires the following tools from the system package manager:\n    autoconf-archive\n    texinfo\nThese can be installed on Ubuntu systems via\n    sudo apt-get update -y\n    sudo apt-get install -y autoconf-archive texinfo\n")
+    message(WARNING "${PORT} currently requires the following packages:\n    autoconf-archive\n    texinfo\nThese can be installed on Ubuntu systems via\n    sudo apt-get update -y\n    sudo apt-get install -y autoconf-archive texinfo\n")
 endif()
 
 set(VERSION 4.1.0)

--- a/ports/mpfr/vcpkg.json
+++ b/ports/mpfr/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "mpfr",
   "version-string": "4.1.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "The MPFR library is a C library for multiple-precision floating-point computations with correct rounding",
   "homepage": "https://www.mpfr.org",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4510,7 +4510,7 @@
     },
     "mpfr": {
       "baseline": "4.1.0",
-      "port-version": 1
+      "port-version": 2
     },
     "mpg123": {
       "baseline": "1.29.2",

--- a/versions/m-/mpfr.json
+++ b/versions/m-/mpfr.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "92e0b9f1e3548a29a47196abda9aeb21bdc17b6a",
+      "version-string": "4.1.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "54544af431b7f178bc7bbe6604f103c40b7aa93b",
       "version-string": "4.1.0",
       "port-version": 1

--- a/versions/m-/mpfr.json
+++ b/versions/m-/mpfr.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "92e0b9f1e3548a29a47196abda9aeb21bdc17b6a",
+      "git-tree": "655c136cd8328e30476ae39a3fd5ff99c915907c",
       "version-string": "4.1.0",
       "port-version": 2
     },

